### PR TITLE
Candacemakedamoore/89 vent plus page

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
OK, this is a placeholder for the ventilator page + some extras

### CODE REVIEW

**Review by wbaccinelli — COMMENTED** [review]
Pay attention to the ids of the elements in `ventilator_match.py`. They should be different from the ones used in other pages, otherwise you will not be able to implement specific callbacks and you will get errors. For example, if you run it now, and navigate to the 'Ventilator match' page, an exception is thrown, because it tries to link the id of one of the elements to a callback written for the 'Preprocessing' page.

**Comment by drcandacemakedamoore** [comment]
OK, added a base page to start, fixed problems I think, please review again @wbaccinelli

File: pages/ventilator_match.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        if active_tab == "original_signal":
+            return dcc.Graph(figure=data["original_signal"])
+        elif active_tab == "adjusted_signal":
+            return dbc.Row(
+                [
+                    dbc.Col(dcc.Graph(figure=data["hist_1"]), width=6),
+                    dbc.Col(dcc.Graph(figure=data["hist_2"]), width=6),
+                ]
+            )
+    return "No tab selected"
```
Comment (wbaccinelli) [inline pages/ventilator_match.py:54]: If we want a change to occur when the user interacts with a graphic element, this need to go through a [callback](https://dash.plotly.com/basic-callbacks). In order to have a more understandable separation, I have been creating a separate script for the callbacks of each page. I suggest to do the same also here.

**Review by wbaccinelli — APPROVED** [review]
Nice! I left a couple of comments. If you want we can go look at them together.

**Comment by drcandacemakedamoore** [comment]
This will be replaced by a new PR

**Pull request closed without merging**